nats/src/main/java/io/micronaut/nats/reactive/PublishState.java

```diff
@@ -1,57 +0,0 @@
-/*
- * Copyright 2017-2020 original authors
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * https://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package io.micronaut.nats.reactive;
-
-import io.micronaut.core.annotation.Nullable;
-
-/**
- * Stores the state of a Nats message to be published.
- * This class should be treated as immutable.
- *
- * @author James Kleeh
- * @since 1.1.0
- */
-public class PublishState {
```

Comment:
Seems this is not annotated with `@Internal` so will need to be kept but deprecated

Reply:
See further discussion